And yet two another:
```
 [ERROR] src/main/java/org/apache/catalina/core/StandardContext.java:[810] (regexp) RegexpSingleline: Line has trailing spaces.
 [ERROR] src/main/java/org/apache/catalina/core/StandardContext.java:[5294] (regexp) RegexpSingleline: Line has trailing spaces.
```

--- a/appserver/web/web-core/src/main/java/org/apache/catalina/core/StandardContext.java
+++ b/appserver/web/web-core/src/main/java/org/apache/catalina/core/StandardContext.java
@@ -807,7 +807,7 @@ public class StandardContext extends ContainerBase implements Context, ServletCo
     protected boolean showArchivedRealPathEnabled = true;
 
     protected int servletReloadCheckSecs = 1;
-    
+
     // Fine tune log levels for ServletContainerInitializerUtil to avoid spurious or too verbose logging
     protected ServletContainerInitializerUtil.LogContext logContext
             = new ServletContainerInitializerUtil.LogContext() {
@@ -5291,7 +5291,7 @@ public class StandardContext extends ContainerBase implements Context, ServletCo
     protected Types getTypes() {
         return null;
     }
-    
+
     protected boolean isStandaloneModule() {
         return true;
     }

--- a/appserver/web/web-glue/src/main/java/com/sun/enterprise/web/WebModule.java
+++ b/appserver/web/web-glue/src/main/java/com/sun/enterprise/web/WebModule.java
@@ -1164,7 +1164,7 @@ public class WebModule extends PwcWebModule implements Context {
     boolean isStandalone() {
         return isStandalone;
     }
-    
+
     protected boolean isStandaloneModule() {
         return isStandalone;
     }

